Update to work with Silverstripe 3.1

timsnadden opened the pull request:
(only tested with LDAP authentication)

timsnadden committed:
Update to work with Silverstripe 3.1 (only tested with LDAP authentication)

timsnadden committed:
Add composer.json

timsnadden committed:
Add _config dir

timsnadden committed:
add type to composer

timsnadden committed:
Add text for tab

timsnadden committed:
Add _config (so that silverstripe knows this is a module)

timsnadden committed:
Remove .gitignore

timsnadden committed:
Add defaults for translated text

camfindlay commented:
@hamishcampbell just checking if you are still an active maintainer of this module? Notice a few pull requests and 3.1 compatibility updates which might be nice to get in :+1:

marijnkampf committed:
Update ExternalAuthenticatedRole.php

Allow editing of user record without getting duplicated (LDAP) identifier error.
Fixed bug in error message on duplicated external anchor.

marijnkampf committed:
Update ExternalAuthenticatedRole.php

Allow editing of user record without getting duplicated (LDAP) identifier error.
Fixed bug in error message on duplicated external anchor.

timsnadden committed:
Use spaces

timsnadden committed:
Merge pull request #1 from marijnkampf/master

Update ExternalAuthenticatedRole.php
